### PR TITLE
Added example of route from kafka to log

### PR DESCRIPTION
**Pull request by vedran-kolka-syntio**
#21 
WIP!

I will add instructions for infrastructure and enrich the readme.

### CODE REVIEW

**Review by oscerd — APPROVED** [review]
Looks good to me and thanks, @davsclaus we could merge this and then try to followup with supporting camel 4.x

**Review by davsclaus — APPROVED** [review]

**Comment by oscerd** [comment]
@vedran-kolka-syntio is this ready?

**Comment by vedran-kolka-syntio** [comment]
No. I marked it as a draft because I haven't tried the terraform script (this is my first, so I have to set up terraform first), and I planned on improving the README.

**Comment by oscerd** [comment]
Ok, thanks.

**Comment by vedran-kolka-syntio** [comment]
I consider the PR ready, so feel free to merge it if you agree.

**Review by oscerd — APPROVED** [review]

File: jbang/azure-eventhubs-kafka-azure-schema-registry/README.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+to solve the instantiation problem, as the class uses a builder for instantiating.
+
+## Build the infrastructure
+
+Choose a globally unique name for the eventhubs namespace and edit it in the terraform [script](main.tf).
+Then, create the services using the script.
+
+## Configure the applications
+
+Use [application.properties.template](application.properties.template) to create `application.properties` and define YOur eventhubs namespace in there.
```
Comment (davsclaus) [inline jbang/azure-eventhubs-kafka-azure-schema-registry/README.md:14]: YOur -> your

File: jbang/azure-eventhubs-kafka-azure-schema-registry/README.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
+# Example for consuming from EventHubs in Avro format, using Azure Schema Registry
```
Comment (davsclaus) [inline jbang/azure-eventhubs-kafka-azure-schema-registry/README.md:1]: Should we say Azure EventHubs ?

**Review by davsclaus — APPROVED** [review]

File: jbang/azure-eventhubs-kafka-azure-schema-registry/README.md (hunk 2 above)
Comment (oscerd) [reply jbang/azure-eventhubs-kafka-azure-schema-registry/README.md:1]: We could fix it later. I think we could merge in this form and then makes some changes.

Comment (davsclaus) [reply jbang/azure-eventhubs-kafka-azure-schema-registry/README.md:1]: ok

**Pull request merged**